tests/test_reader.py

```diff
@@ -222,30 +222,16 @@ def test_get_images(src, expected_images):
 
     src_abs = RESOURCE_ROOT / src
     reader = PdfReader(src_abs)
-
-    with pytest.raises(TypeError):
-        page = reader.pages["0"]
-
-    page = reader.pages[-1]
```

Comment:
Do we test this somewhere and what should be the expected result? Probably return the last page?

Reply:
Afaik, it is not tested that indeed the last page is returned. I might be wrong, but to me it seems it was put here to see that `__getitem__` behaves well. Together with the `TypeError` check, it was confusing me why these tests were made here and not in a separate function - note that the `TypeError `check I moved now into test_page.py.

Reply:
I do not want to remove any tests which could be of value by this, thus I would prefer if we would move this line to the correct test file and test method as well.

Reply:
I would prefer a dedicated function `test_negative_index` or to rename the current function to `test_page_indices`.